src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -156,17 +121,45 @@ const ColonyHome = ({ match, location }: Props) => {
         <div className={styles.mainContent}>
           <ColonyTotalFunds colony={colony} />
           <Switch>
-            <Route path={COLONY_EVENTS_ROUTE} component={() => <>Events</>} />
             <Route
+              exact
+              path={COLONY_HOME_ROUTE}
+              component={() => (
```

Comment:
This is the wrong component here, it should be just `() => <>Actions</>` as #2287 adds in the required content here.

I thought the home looked a bit _too_ wonky:

![Screenshot from 2020-11-12 16-00-26](https://user-images.githubusercontent.com/1193222/98949351-443ce000-2500-11eb-8d3b-9394262223d6.png)